plugins/joblib_launcher/example/conf/hydra/launcher/example.yaml

```diff
@@ -0,0 +1,5 @@
+hydra:
+  launcher:
+    class: hydra_plugins.joblib_launcher.JoblibLauncher
+    params:
+      n_jobs: -1
```

Comment:
The purpose of JoblibLauncherSearchPathPlugin is to allow user code to find the configuration packaged by the plugin.
There is no need to include this configuration file with the example.